src/components/Table/GridTable.tsx

```diff
@@ -556,12 +557,12 @@ function renderVirtual<R extends Kinded>(
 
         // If it is a chrome row, then we are not using `display: contents;`, return the height of this element.
         if ("chrome" in maybeContentsDiv.dataset) {
-          return maybeContentsDiv.offsetHeight;
+          return maybeContentsDiv.getBoundingClientRect().height || 1;
```

Comment:
Just curious, but what's better about `getBoundingClientRect()` vs `offsetHeight`? Just more accurate?

Reply:
Ah gotcha, gotcha, I read the PR description; makes sense, I think this is great as-is then, i.e. not need for a follow up in-source comment.